src/lib/flow/codegen/block2ast.ts

```
@@ -651,26 +652,50 @@ export class CodeGenASTGenerator {
+					interpolated: value.interpolated
 				};
 			case TypedInputEnum.Number:
 				return {
 					type: ITypedInputEnum.Number,
 					value: value.value
 				};
 			case TypedInputEnum.Table:
+				let tableValue: ITypedInputTableEntry[] = value.value.map((entry) => ({
+					key: this.visitTypedInput(entry.key),
+					value: this.visitTypedInput(entry.value)
+				}));
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)